database/migrations/2020_04_26_215133_create_addresses_table.php

```diff
@@ -17,7 +17,8 @@ public function up()
             $table->unsignedBigInteger('contact_id');
             $table->unsignedBigInteger('address_type_id')->nullable();
             $table->boolean('is_past_address')->default(false);
-            $table->string('street')->nullable();
+            $table->string('line_1')->nullable();
+            $table->string('line_2')->nullable();
```

Comment:
We shouldn't change this migration since it will not work with people who already have Chandler already. We should create a new migration instead, example here: https://github.com/monicahq/chandler/blob/main/database/migrations/2022_11_01_174411_add_religion_to_contact.php